### ISSUE TITLE
Is it possible to turn off debug logging for `provider-azure-installer` and `liveness-probe` containers please

### ISSUE THREAD

**Issue by ppanyukov** [Azure/secrets-store-csi-driver-provider-azure#157]
(Not sure if this belongs here, or in the dependecy repo `secrets-store-csi-driver`.)

Using helm chart: https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/0.0.6/charts

We have two containers which are quite chatty in debug logging, and this costs us money as we ship all these logs to Azure Monitor / Log Analytics and they charge us per byte :) 

What I'd like to have is to turn this debug logging off as it doesn't seem to be of any value (unless I'm mistaken?).

Two containers:

* `csi-secrets-store-provider-azure-secrets-store-csi-driver / liveness-probe`

Logs are like this, every 30 seconds:

```
I0715 13:18:36.320788       1 main.go:53] Sending probe request to CSI driver "secrets-store.csi.k8s.io"
I0715 13:18:36.321650       1 main.go:71] Health check succeeded
I0715 13:18:51.320813       1 main.go:53] Sending probe request to CSI driver "secrets-store.csi.k8s.io"
I0715 13:18:51.321387       1 main.go:71] Health check succeeded
```

* `csi-secrets-store-provider-azure / provider-azure-installer`

Logs are like this, one line per minute:

```
install done, daemonset sleeping
install done, daemonset sleeping
install done, daemonset sleeping
install done, daemonset sleeping
install done, daemonset sleeping
```

I am using `secrets-store-csi-driver.logLevel.debug = false` when deploying the chart but it does not affect these two bits.

It would be nice if we could silence such debug output somehow properly.

If there are any workarounds, that would be good to know too.

**Comment by ppanyukov** [comment]
In terms of data volume and cost, if I trust Log Analytics and my math skills, we have this:

```
liveness-probe: 120MB per node per month == £0.25 per node per month
provider-azure-installer: 12MB per node per month == £0.025 per node per month
```

I know, not massive amounts but things add up :)

**Comment by aramase** [comment]
@ppanyukov I've opened a PR in the driver to bump the `livenessprobe` container version to `v2.0.0` in which they have bumped the log level for those frequent logs to a higher value. With `v1.1.0` the logs are printed by default and there is no way to configure the verbosity. 

For the `provider-azure-installer`, we can certainly remove the periodic logs in the next release. I've opened https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/158 to track that change.
